Main.py

```diff
@@ -34,6 +34,7 @@ def main(args, seed=None, baked_server_options: Optional[Dict[str, object]] = No
     multiworld = MultiWorld(args.multi)
 
     logger = logging.getLogger()
+
```

Comment:
```suggestion
```
Just looks like an accidental newline

Reply:
It is and isn't. Originally there were other lines here so i separated the logging instantiation with an extra blank line because setting up the multiworld seems like a logically different thing from instantiating logging. If you look at this from a wider lens this logging instantiation is smack in the middle of the multiworld instantiation/setup, which is a little awkward... imho it should move up slightly (just below the assertion, maybe?) so that it's not in the middle of the multiworld setup logical block (and, imho, there should be a blank line above the multiworld setup comment to start that logical block; let it breath a little).

So not taking this newlilne out was a bit of a mistake (i'll remove it), but i think it somewhat calls out that the code could be rearranged slightly to be a bit more clear.